pkg/provisioning/cbnt/tools.go

```diff
@@ -125,7 +125,7 @@ func ParseFITEntries(image []byte) (bpm *fit.EntryBootPolicyManifestRecord, km *
 		}
 	}
 	if bpm == nil || km == nil || acm == nil {
-		return nil, nil, nil, fmt.Errorf("image has no BPM (isNil:%v) or/and KM (isNil:%v) or/and ACM (isNil:%v)", bpm == nil, km == nil, acm == nil)
```

Comment:
Nit:
It is not common to get an error and valid data...
I would rather remove error as a part of function's result completely (in this case we miss possible handling of the error) or do not return error on partial results (remove lines 127-129)

Reply:
I agree. I had the same thoughts, but I did want to focus on the main task instead of polishing the code here.